lib/lita/handlers/totems.rb

```diff
@@ -1,10 +1,15 @@
 require 'lita'
 require 'chronic_duration'
 require 'redis-semaphore'
+require 'signalfx'
 
 module Lita
   module Handlers
     class Totems < Handler
+      
+      def signalfx_client 
+        @signalfx_client ||= SignalFx.new ENV['LITA_SIGNALFX_TOKEN']
```

Comment:
Should we be using an instance variable + a method? It looks like we only reference the method elsewhere. Could we move setting the instance variable to the constructor and then reference the instance variable where necessary?

Reply:
I think it maybe better to pull out this logic from this class altogether though. See: https://github.com/change/lita-totems/pull/2/files#r826453118